modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenuSmoke.testcase

```diff
@@ -0,0 +1,141 @@
+@component-name = "portal-platform-experience-and-design-system"
+definition {
+
+	property osgi.modules.includes = "accessibility-menu-web";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.component.names = "Accessibility Menu";
+	property testray.main.component.name = "Accessibility Menu";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		TestUtils.hardRefresh();
+
+		task ("Given user with admin privileges") {
+			User.firstLoginPG();
+		}
+	}
+
+	tearDown {
+		var testLiferayVirtualInstance = PropsUtil.get("test.liferay.virtual.instance");
+
+		if (${testLiferayVirtualInstance} == "true") {
+			PortalInstances.tearDownCP();
+
+			TestUtils.hardRefresh();
+
+			HeadlessPortalInstanceAPI.cleanAllPortalInstances();
+		}
+		else {
+			TestUtils.hardRefresh();
+		}
+	}
+
+	@description = "LPS-178192. Verifies only if the setting has never been changed by the user, either when signed in or not signed in, will the default value be displayed.."
+	@priority = 5
+	test BrowserLocalStorageSmoke {
+		property portal.acceptance = "true";
+
+		task ("Given created new user A") {
+			JSONUser.addUser(
+				userEmailAddress = "userea@liferay.com",
+				userFirstName = "userfnA",
+				userLastName = "userlnA",
+				userScreenName = "usersnA");
+		}
+
+		task ("And signed out as guest user") {
+			SignOut.signOut();
+		}
+
+		task ("When the toggle shows underline to ON state") {
+			AccessibilityMenu.enableSpecificOption(key_toggleSwitchLabel = "Underlined Links");
+
+			AssertChecked.assertCheckedNotVisible(
+				key_toggleSwitchLabel = "Underlined Links",
+				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
```

Comment:
please remove this assertion. No assertions in Given and When steps as much as possible